jormungandr-integration-tests/src/common/configuration/node_config_model.rs

```diff
@@ -5,9 +5,9 @@ use self::serde_derive::{Deserialize, Serialize};
 use super::file_utils;
 use std::path::PathBuf;
 #[derive(Debug, Serialize, Deserialize, Clone)]
-pub struct Logger {
-    pub verbosity: i32,
-    pub format: String,
+pub struct Log {
+    pub level: Option<String>,
```

Comment:
Can't be `slog::Level`?

Reply:
These are test tools designed by Darek, they are all String- and int-based. I didn't want to mess with this strategy.